### PR TITLE
1.2.2

### PR DESCRIPTION
**Pull request by leewang0**
## Description of the change

Removed enum values for 

* `SyncStatus.status`
* `BankInfo.account_type`
* `AccountDetailsAndActionsIntegrationSerializer.categories`
* `AccountDetailsAndActionsSerializer.category`
* `AccountDetailsAndActionsSerializer.status`
* `AccountDetailsAndActionsSerializer.method`
* `AccountDetailsAndActionsSerializer.requestFormat`

added BankInfo endpoints

added new write endpoint for TimeOffRequest

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> All PRs at Merge must be backed by an Asana ticket. Create one here, and then replace this line with the link. https://app.asana.com/0/1198991781422585/list

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests, or the description above explains how testing was performed

### Code review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or gif attached. Download "Kap" from the Mac App store for easy gif screen capture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +41,19 @@ Add this dependency to your project's POM:
...
 ```
 
+### Gradle users
+
+Add this dependency to your project's build file:
+
+```groovy
+compile "dev.merge.hris:merge-hris-client:1.2.2"
+```
+
```
Comment (dan-merge) [inline README.md:56]: ^should probably remove this?

File: README.md [hunk 2]
```diff
@@ -131,49 +139,49 @@ Class | Method | HTTP request | Description
 *LinkedAccountsApi* | [**linkedAccountsList**](docs/LinkedAccountsApi.md#linkedAccountsList) | **GET** /linked-accounts | 
 *LocationsApi* | [**locationsList**](docs/LocationsApi.md#locationsList) | **GET** /locations | 
 *LocationsApi* | [**locationsRetrieve**](docs/LocationsApi.md#locationsRetrieve) | **GET** /locations/{id} | 
-*PassthroughApi* | [**passthroughCreate**](docs/PassthroughApi.md#passthroughCreate) | **POST** /passthrough |
-*PayrollRunsApi* | [**payrollRunsList**](docs/PayrollRunsApi.md#payrollRunsList) | **GET** /payroll-runs |
-*PayrollRunsApi* | [**payrollRunsRetrieve**](docs/PayrollRunsApi.md#payrollRunsRetrieve) | **GET** /payroll-runs/{id} |
```
Comment (dan-merge) [inline README.md:136]: why does this remove payroll runs?

Comment (leewang0) [reply README.md:136]: Great catch, i've re-added those (they were removed temporarily on the david/java-sdk branch of merge-backend but https://github.com/merge-api/merge-backend/pull/3235 fixed it.

File: README.md (hunk 1 above)
Comment (leewang0) [reply README.md:56]: I don't think we should, we may compile this java package with `maven` but `gradle` is a very popular alternative build tool that many people consider superior (it is newer and has some extra features). It is backwards compatible, so our pom.xml should still work with it. I don't see the harm?

**Review by dan-merge — APPROVED** [review]

**Pull request merged**